operators/vision/draw_bounding_box.hpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ocos.h"
+#include "string_utils.h"
+
+#include <cstdint>
+
+namespace ort_extensions {
+struct DrawBoundingBox : BaseKernel {
+  DrawBoundingBox(const OrtApi& api, const OrtKernelInfo& info) : BaseKernel(api, info) {
+    thickness_ = TryToGetAttributeWithDefault<int64_t>("thickness", 4);
+    num_classes_ = static_cast<int32_t>(TryToGetAttributeWithDefault<int64_t>("num_classes", 10));
+    auto need_sort = TryToGetAttributeWithDefault<int64_t>("sort_by_score", 0);
```

Comment:
If we don't sort by score, we're not picking the top x classes/results which seems like an unexpected way to output results.

Reply:
If we have a few class, so it doesn't make sense to sort classes right?  For example, the first class has 7 object, and the second has 4 object, we can't say the first class is priority over the second.

Reply:
Got it. Will do the sort before drawing boxes.